.github/workflows/cd.yml

```
@@ -11,6 +11,98 @@ permissions:
+      - name: Build and push image
+        uses: docker/build-push-action@471d1dc4e07e5cdedd4c2171150001c434f0b7a4 # v6.15.0
+        with:
+          context: .
+          file: ./Dockerfile
+          push: true
+          tags: ${{ steps.meta-server.outputs.tags }}
+          labels: ${{ steps.meta-server.outputs.labels }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+  deploy-staging:
```

actionlint (1.7.10)

[error] job "deploy-staging" needs job "build-and-push" which does not exist in this workflow

(job-needs)